cli.sh

```diff
@@ -124,7 +124,11 @@ function sshkeyadd () {
 }
 
 function version {
-  npm info '@treehouses/cli' version
+  installed_cli_version=$(node -p "require('/usr/lib/node_modules/@treehouses/cli/package.json').version")
```

Comment:
I think this should not be hardcoded
we could use the same as the init of the script
```bash
SCRIPTPATH=$(realpath "$0")
SCRIPTFOLDER=$(dirname "$SCRIPTPATH")
installed_cli_version=$(node -p "require('$SCRIPTFOLDER/package.json').version")
```

Reply:
@i5o Thank you for the review. I was not sure about the hard coded path. Didn't realize there's already something I could use at the top.